splinterd/src/daemon.rs

```diff
@@ -289,6 +289,8 @@ impl SplinterDaemon {
                 })?;
         let orchestrator = ServiceOrchestrator::new(
             vec![Box::new(ScabbardFactory::new(
+                None,
```

Comment:
These options feel like they should probably be a single config object: reading this is pretty heavy on the `None`'s